packages/controller/buidler.config.js

```diff
@@ -0,0 +1,48 @@
+const { usePlugin } = require('@nomiclabs/buidler/config')
+
+usePlugin("@nomiclabs/buidler-ganache")
+usePlugin('@nomiclabs/buidler-truffle5')
+
+module.exports = {
+  networks: {
+    // Local development network using ganache. You can set any of the
+    // Ganache's options. All of them are supported, with the exception
+    // of accounts.
+    // https://github.com/trufflesuite/ganache-core#options
+    ganache: {
+      url: 'http://localhost:8545',
+      gasLimit: 6000000000,
+      defaultBalanceEther: 100
+    },
+    // Mainnet network configured with Aragon node.
+    mainnet: {
+      url: 'https://mainnet.eth.aragon.network',
+      accounts: [
+        process.env.ETH_KEY ||
+          '0xa8a54b2d8197bc0b19bb8a084031be71835580a01e70a45a13babd16c9bc1563',
```

Comment:
Do we need this here? Maybe it would be good to add a comment saying that this corresponds to the first address of the nmemonic we use in aragon devchain, so nobody gets scared ;-)

Reply:
Haha, will do!

Reply:
Done in https://github.com/aragon/aragon-network-token/pull/19.